assets/stylesheets/openqa.scss

```diff
@@ -200,15 +200,24 @@ ul.modcategory {
 }
 
 .no_hover:hover {
-   text-decoration: none;
+    text-decoration: none;
 }
 
 .candidate_needles {
     margin-right: 1em;
 }
 
 .help_popover  {
-    margin-left: 1em;
+    margin: 0px 5px;
```

Comment:
But making this dependent on the font size the element has doesn't seem correct, too. I mean when increasing/decreasing the font size, you don't want necessarily also increase/decreasing the margin, right?